src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -42,6 +42,8 @@
 
 public class Clone extends VSphereBuildStep {
 
+	private final int TIMEOUT_DEFAULT = 60;
```

Comment:
Indentation incorrect - use spaces, not tabs (tabs are evil - they never display the same as some people set them to 4, some 2, some 8 etc).